conn_query.go

```diff
@@ -80,6 +79,8 @@ func (c *connect) query(ctx context.Context, release nativeTransportRelease, que
 		}
 		close(stream)
 		close(errors)
+
+		c.conn.SetDeadline(time.Time{})
```

Comment:
I think we might need to do this for `SetReadDeadline` too. As the SetDeadline is an override on the context. I am unsure what we're using exactly in our case.

p.s. thanks for jumping on this so quickly!